code/processes/wdb.q

```diff
@@ -226,7 +226,7 @@ d:()!()
 doreload:{[pt]
 	.wdb.reloadcomplete:0b;
 	/-inform gateway of reload start
-	informgateway["reloadstart[]"];
+	informgateway(`reloadstart;);
```

Comment:
Typo, as per both comments this should be ``(`reloadstart;`)``.

Reply:
That was foolish of me